Products/CMFPlone/controlpanel/browser/usergroups_groupdetails.py

```diff
@@ -2,13 +2,23 @@
 from plone.base import PloneMessageFactory as _
 from plone.protect import CheckAuthenticator
 from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.controlpanel.browser.usergroups import (
-    UsersGroupsControlPanelView,
-)
+from Products.CMFPlone.controlpanel.browser.usergroups import UsersGroupsControlPanelView
+from Products.CMFPlone.PloneTool import PloneTool
 from Products.statusmessages.interfaces import IStatusMessage
 
-
 class GroupDetailsControlPanel(UsersGroupsControlPanelView):
+    def extract_invalid_emails(self, emails):
+        """Validate a list of email addresses."""
+        if isinstance(emails, str):
+            emails = [e.strip() for e in emails.split('\n') if e.strip()]
+        
+        invalid_emails = []
+        for email in emails:
+            if not PloneTool.validateEmailAddresses(email):
```

Comment:
This code will break because you are trying to call the method without initializing the class:
```suggestion
            if not plone_utils.validateEmailAddresses(email):
```

But anyway this will not be working because it expects a list:
```
    @security.public
    def validateEmailAddresses(self, addresses):
        # Validate a list of possibly several email addresses, see also
        # validateSingleEmailAddress.
        if not isinstance(addresses, str):
            return False
```

Writing a test would have helped you to debug this issues earlier. I suggest you to do that!